application/src/View/Helper/SearchFilters.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace Omeka\View\Helper;
+
+use Zend\ServiceManager\ServiceLocatorInterface;
+use Zend\View\Helper\AbstractHelper;
+
+class SearchFilters extends AbstractHelper
+{
+    /**
+     * The default partial view script.
+     */
+    const PARTIAL_NAME = 'common/search-filters';
+
+    /**
+     * Construct the helper.
+     *
+     * @param ServiceLocatorInterface $serviceLocator
+     */
+    public function __construct(ServiceLocatorInterface $serviceLocator)
+    {
+        $this->request = $serviceLocator->get('Request');
+    }
+
+    /**
+     * Render filters from search query.
+     *
+     * @return array
+     */
+    function __invoke($partialName = null) 
+    {
+        $partialName = $partialName ?: self::PARTIAL_NAME;
+
+        $escape = $this->getView()->plugin('escapeHtml');
+
+        $filters = array();
+        $exclude = array('submit', 'page', 'sort_by', 'sort_order');
+        $api = $this->getView()->api();
+        $query = $this->request->getQuery()->toArray();
+        $queryTypes = array(
+            'eq' => $escape('has exact value(s)'),
```

Comment:
All these uses of escape around static strings should be translate calls instead.

The stuff that needs to be escaped is the user input from the request and the stuff coming from the DB (labels of things, etc.), but it's probably easier to just escape that stuff as you're outputting it in the partial.